### PR TITLE
Add void-docs man page and building information.

### PR DESCRIPTION
**Pull request by ericonr**
Related to the void-docs package.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+fd -t d "" ./ -x mkdir -p "../mandoc/{}"
+
+fd "\.md" ./ -x pandoc \
+    -V "title={/.}" -V "section=7" -V "header=Void Docs" -s \
+    -o "../mandoc/{.}.7" "{}"
+
+cd -
+
+# Buld reference man page
```
Comment (flexibeast) [inline build.sh:20]: "Build"

File: void-docs.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+$ less /usr/share/doc/void/markdown/SUMMARY.md
+```
+
+Viewing the Kernel page of the markdown documentation with **mdcat**:
+
+```
+$ mdcat /usr/share/doc/void/markdown/config/kernel.md
+```
+
+Viewing the Cron page of the roff documentation with **mandoc(1)**:
```
Comment (flexibeast) [inline void-docs.md:54]: "cron"?

File: void-docs.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+# AVAILABILITY
+
+This man page is part of the void-docs package and is available from
+https://github.com/void-linux/void-docs.
+
+# BUGS
+
+The Void Linux documentation tries to limit itself to content that is specific
+to the distro. Therefore, if you feel something is missing, it might have been
```
Comment (flexibeast) [inline void-docs.md:68]: s/the distro/Void/.

File: void-docs.md (hunk 2 above)
Comment (ericonr) [reply void-docs.md:54]: I was going by the page title in this one. Perhaps put it in quotes anyway?

File: void-docs.md (hunk 3 above)
Comment (ericonr) [reply void-docs.md:68]: Much better

File: void-docs.md (hunk 2 above)
Comment (flexibeast) [reply void-docs.md:54]: Ah, yeah, sure.

Comment (ericonr) [reply void-docs.md:54]: Fixed, then :p

**Comment by ericonr** [comment]
@flexibeast good to go?

**Pull request merged**